### DISCUSSION TITLE
Is gemini-pro supported?

### DISCUSSION THREAD

**Discussion by rosmur** [BerriAI/litellm#1238] [Q&A] [answered]
Are the new models from Google, gemini-pro, gemini-ultra supported as yet? or any roadmap for support?

**Comment by ishaan-jaff** [comment, marked as answer]
supported here: https://docs.litellm.ai/docs/providers/vertex @rosmur

**Comment by rosmur** [comment]
sweet! didnt catch that. thanks!

**Reply by ishaan-jaff** [reply]
where was it unclear @rosmur ? I'll add some docs about gemini there

**Reply by rosmur** [reply]
Looks like "[Gemini]" in brackets is added to the google vertex line in the table on the Read Me :) Now its very clear!

**Comment by ishaan-jaff** [comment]
@rosmur how do you use the litellm proxy server ?
